### PR TITLE
Update website content

### PR DESCRIPTION
**Pull request by thepetk**
Updates programme content, removes the community voting deadline and adds a sponsor one

### CODE REVIEW

**Review by cmaureir — APPROVED** [review]

File: src/content/deadlines/00_cfp.mdx [hunk 1]
```diff
@@ -1,10 +0,0 @@
----
```
Comment (patrick91) [inline src/content/deadlines/00_cfp.mdx:1]: did you delete this because it shows on the home page? 😊

Comment (thepetk) [reply src/content/deadlines/00_cfp.mdx:1]: yeap I'd like to remove that in favor of sponsors. Preview was ok but let me know if I have to update anything else!

Comment (patrick91) [reply src/content/deadlines/00_cfp.mdx]: I've done that in code, so we can keep the deadlines, and only show the last 3 😊

**Comment by thepetk** [comment]
Done!

**Pull request merged**